### DISCUSSION TITLE
Youtube TV

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#346] [General]
Anyone watch it?
What is your opinion, worth buying the subscriptions ?

**Comment by XelaAl** [comment]
YouTube TV is a great money-saving solution because you get all of these things for $65 per month: more than 85 channels. NFL Network, NBA TV, and MLB Network.

**Comment by hernessteve** [comment]
Youtube is not bad but you might wanna check few good alternatives as well - [https://www.firesticktricks.com/youtube-tv-alternatives.html)https://www.firesticktricks.com/youtube-tv-alternatives.html](url). I suggest you try few different ones and decide which one is worth your time. They are different and all with the pros and cons.
